pkg/build/controller/buildconfig/buildconfig_controller.go

```diff
@@ -174,9 +175,52 @@ func (c *BuildConfigController) buildConfigAdded(obj interface{}) {
 // buildconfig is updated or there is a relist of buildconfigs
 func (c *BuildConfigController) buildConfigUpdated(old, cur interface{}) {
 	bc := cur.(*buildv1.BuildConfig)
+	oldBuildConfig := old.(*buildv1.BuildConfig)
+	// TODO: Remove the event after the image trigger behavior is removed.
```

Comment:
nit: reference @otaviof 's JIRA in this comment

Reply:
done - referenced the issue where we are going to actually remove this behavior.